gui/src/granite/GraniteWizard.tsx

```diff
@@ -922,8 +843,14 @@ function getRequiredSpace(
       ? DEFAULT_MODEL_GRANITE_LARGE
       : DEFAULT_MODEL_GRANITE_SMALL;
   const models: string[] = [graniteModel.model, "nomic-embed-text:latest"];
-  return models
-    .filter((model) => statusByModel.get(model) === ModelStatus.missing)
+  let missingModels = models
+    .filter((model) => statusByModel.get(model) !== ModelStatus.installed);
+
+  if (missingModels.length === 0) {
```

Comment:
Could use a comment:
  // If the user clicks back to a skipped download step, we want to show entire download size, not 0B